cmake/JSSConfig.cmake

```diff
@@ -1,3 +1,10 @@
+##
+# NSS V3.112 have some PQC defs in auth_alg_defs of ssl3con.c
+# that need to be reflected into JSS SSLCipher.c (auth_alg_defs[])
+# The upstream NSS (currently v3.113) apparently has no such defs.
+#
+option(ENABLE_NSS_VERSION_PQC_DEF "Enable PQC DEF to match NSS" OFF)
```

Comment:
Usually the options are defined in the top-level `CMakeLists.txt`.

Reply:
> Usually the options are defined in the top-level `CMakeLists.txt`.

OK, I will move the Option back to CMakeLists.txt.  That's where I put it yesterday.